### ISSUE TITLE
Can't add pixelfed.social acount

### ISSUE THREAD

**Issue by arkydon** [mastodon/mastodon-android#818]
If I try to add a pixelfed.social account I get this error when I try to authorise Mastodon:  "Required field 'Scope' of type string was null in token". I have changed the default browser to other browsers and still get this same error.

![Screenshot_20240329-193953_Mastodon.png](https://github.com/mastodon/mastodon-android/assets/7901865/c861780a-da3e-4fb7-9754-1e3a040c1743)

**Comment by grishka** [comment]
Non-Mastodon servers *may* work depending on how compatible their API is, but are not officially supported.
